osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableSwell.cs

```diff
@@ -237,7 +237,7 @@ protected override void UpdateStateTransforms(ArmedState state)
 
                 case ArmedState.Miss:
                 case ArmedState.Hit:
-                    using (BeginAbsoluteSequence(Time.Current, true))
+                    using (BeginDelayedSequence(HitObject.Duration, true))
```

Comment:
I can't seem to see this causing any change in behaviour, but it kind of looks like it would - what am I missing?

Reply:
Didn't really mean to bundle this with this PR. The old code would actually work incorrectly if a large time jump occurred without the presence of a `FrameStabilityContainer`, since it would base the animations on the new time rather than the point in time that the hit should have occurred (`HitObject.StartTime + HitObject.Duration`).